packages/app-extensions/src/formField/defaultMapping.js

```diff
@@ -32,8 +36,10 @@ export default {
     'date': editableValueFactory('date-range'),
     'birthdate': editableValueFactory('date-range')
   },
+  'remote-field': editableValueFactory('remote'),
```

Comment:
haven't we found out that this is not needed?

Reply:
Yes.